src/main/java/webDriver/GooglePage.java

```diff
@@ -0,0 +1,33 @@
+package webDriver;
+
+import org.openqa.selenium.*;
+
+public class GooglePage {
+    private By searchField = By.id("lst-ib");
+    private WebDriver driver;
+
+    public GooglePage(WebDriver driver) {
+        this.driver = driver;
```

Comment:
Use PageFactory for initializing elements. Usually it invokes in constructor as follows `PageFactory.initElements(driver, this);` Then you can use @FindBy annotation